core-processor/src/ext.rs

```diff
@@ -487,6 +487,7 @@ impl<'a, LP: LazyPagesInterface> ExtMutator<'a, LP> {
                 Ok(packet.gas_limit().unwrap_or(0))
             }
             DispatchKind::Signal => unreachable!("Signals can't be sent as a syscall"),
+            _ => unreachable!("Multiple dispatch kind is impossible here"),
```

Comment:
Consider to use `enumflags2` create to clearly distinguish `DispatchKind` and set of kinds

Reply:
I switched to enumflags2. It turned out not very nice because it is impossible to apply TypeInfo trait on BitFlags<T>.